app/help/en/gateways/ioxbank.md

```diff
@@ -0,0 +1,42 @@
+# ioxbank Instant Swap Service
+A Bridge Instant Swap Service is responsible for moving cryptocurrencies between BitShares and other Blockchains. You can easily identify those cryptocurrencies supported by ioxbank, they are prefixed with **IOB.** and marked as featured in dashboard page.
+
+## Help Docs
+[https://support.ioxbank.com/docs](https://support.ioxbank.com/docs)
+
+## Support
+- [Live Support](https://t.me/ioxbank)
+- [Open a Ticket](https://support.ioxbank.com)
+
+## IOB.XRP features
+- Instant swap
+- Min. amount: 20 XRP
+- No KYC
+- No limits
+- 0/0.1% market fee for maker/taker
+- Low withdraw fee
+
+### Gateway Transparency Information
+- [XRPL Balance](https://livenet.xrpl.org/accounts/rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y)
+- [IOB.XRP Current supply](/asset/IOB.XRP)
+
+### Deposit to your BitShares wallet from your XRP wallet
+- **XRP Destination address:** rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y
```

Comment:
Leave this to the deposit modal which generates the address on the fly from your API. If API not available, no adress. Otherwise users might use this manually and end up getting stuck, and also you might want to dynamically change this adress without having to wait for release

Reply:
There is no address generation happening hence there is no need to use the API to show your tag; tags doesn't need to be generated, All numbers are there, they are just linked to every user id over BitShares; this XRP address is dedicated for BitShares Blockchain and wouldn't need to be changed.

Reply:
ioxbank is offering the ability for deep integration between (BitShares Blockchain and XRP Ledger); ANY one with addresses over both the Blockchains can integrate with the instant swap service in order to swap XRP assets to BitShares assets instantly; hence at this phase of integration; users of XRP Ledger can use ioxbank to send XRP (core asset of XRP Ledger) to ANY address over BitShares blockchain and users of BitShares Blockchain can use ioxbank to send IOB.XRP (Pegged Issued Asset) to ANY XRP Ledger address.

Reply:
The address "rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y" is loaded from your gateway API when requesting deposit, is it not? I have not seen this hardcoded in the UI deposit modal code?

Reply:
Yes it's not hardcoded within the UI but it's a static address over the API and it will not be changed.

Please refer to address info: https://livenet.xrpl.org/accounts/rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y as this address belongs to the domain ioxbank.com

Reply:
My point is: If you ever need to change the adress for whatever reason, or disable deposits you can do that. If you hardcode the address anywhere, you can't, or shouldn't.

In any case, I guess this is within the responsibility of you as the operator, so if you want to keep it that way, it's fine. I do recommend against hardcoding though

Reply:
I will keep it this way; thanks.